scania-digital-design-system/tegel
# feat(banner): removal of persistent prop

**Describe pull-request**  
Removing persistent prop

**Solving issue**  
Part of Angular/React clean-up

**How to test**  
1. Go to the Storybook PR preview link
2. Open the Banner component
3. There should not be "Disable closing" control anymore
4. "persistent" prop is removed from "Notes" tab of Banner component too
 
**Suggested test steps**

**Additional context**  
Compare it with the [production version of Storybook. 
](https://tegel-storybook.netlify.app/?path=/story/components-banner--default)
